sources/us/md/statewide.json

```diff
@@ -7,7 +7,7 @@
         "country": "us",
         "state": "md"
     },
-    "note": "Does not cover the entire state. Referred to link from https://www.arcgis.com/home/item.html?id=20878dd2c20949a8a14a5a282ab98807",
+    "note": "Does not cover the entire state. Referred to link from https://www.arcgis.com/home/item.html?id=20878dd2c20949a8a14a5a282ab98807. Also note that about 25,000 addresses is incorrectly positioned at -81.5291885,37.5772618 (you might want to filter those out)",
```

Comment:
That's the "Null Island" of their state plane system – that's likely to happen in other datasets, too. Would be amusing to set an (optional) null island coord for each dataset and filter those automatically, or tag them as "visiting Null Island" in export.